core/src/main/java/org/axonframework/commandhandling/disruptor/CommandHandlerInvoker.java

```diff
@@ -284,15 +285,22 @@ private void removeFromCache(String aggregateIdentifier) {
 
         @Override
         public void send(Message<?> message, ScopeDescriptor scopeDescription) throws Exception {
-            if (scopeDescription instanceof AggregateDescriptor) {
-                load(((AggregateDescriptor) scopeDescription).getIdentifier().toString()).handle(message);
+            if (scopeDescription instanceof AggregateScopeDescriptor) {
+                String aggregateIdentifier = ((AggregateScopeDescriptor) scopeDescription).getIdentifier().toString();
+                Aggregate<T> aggregate = load(aggregateIdentifier);
+                if (aggregate != null) {
```

Comment:
This is always non-null. `load()` throws an `AggregateNotFoundException` when an aggregate can't be found.